ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/events/CommitTxnHandler.java

```diff
@@ -107,12 +107,33 @@ public void handle(Context withinContext) throws Exception {
 
     if (!withinContext.hiveConf.getBoolVar(HiveConf.ConfVars.REPL_DUMP_METADATA_ONLY)) {
 
+      boolean replicatingAcidEvents = true;
+      if (withinContext.hiveConf.getBoolVar(HiveConf.ConfVars.REPL_BOOTSTRAP_ACID_TABLES)) {
+        // We do not dump ACID table related events when taking a bootstrap dump of ACID tables as
+        // part of an incremental dump. So we shouldn't be dumping any changes to ACID table as
+        // part of the commit. At the same time we need to dump the commit transaction event so
+        // that replication can end a transaction opened when replaying open transaction event.
+        LOG.info("writeEventsInfoList will be removed from commit message because we are " +
```

Comment:
Can be debug log as it is not so useful for user.

Reply:
Done.